x/staging/posts/types/keys.go

```diff
@@ -84,9 +84,14 @@ func PostReactionsStoreKey(id string) []byte {
 	return append(PostReactionsStorePrefix, []byte(id)...)
 }
 
-// ReactionsStoreKey turns the combination of shortCode and subspace to a key used to store a reaction into the reaction's store
-func ReactionsStoreKey(shortCode, subspace string) []byte {
-	return append(ReactionsStorePrefix, []byte(shortCode+subspace)...)
+// ReactionSubspacePrefix returns the prefix used to store all the reactions for the subspace having the given id
+func ReactionSubspacePrefix(subspace string) []byte {
```

Comment:
To make this clear, since we have posts reactions and registered reaction I think we should rename this to be `RegisteredReactionsPrefix` instead